### ISSUE TITLE
【FR】Bug: FlagGems linear/softmax 在 Ascend910 输出乱码 (RekaAI/reka-flash-3)

### ISSUE THREAD

**Issue by Lxiparer** [FlagOpen/FlagGems#6109]
### 模型
RekaAI/reka-flash-3

### 崩溃类型
`FlagGems` `vllm-plugin-FL`

### 根因定位
代码注入模式（`--mode flagos`）绕过 plugin dispatch，导致 FlagGems `linear` 和 `softmax` 算子在 Ascend910 输出全 `!` 乱码。

### 涉及 FlagGems 算子
- `linear`
- `softmax`

### 涉及 Plugin 组件
`ascend.yaml flagos_blacklist` — 代码注入模式绕过了 plugin 的黑名单配置

### Environment
| Item | Value |
|------|-------|
| Hardware | 华为(Ascend) |
| GPU | 910C |
| vLLM | 0.20.2 |
| vllm-plugin-FL | 0.2.0 |
| FlagGems | 5.3.4 |
| PyTorch | 2.10.0+cpu |

### 启动崩溃日志
```
[2026-09-08 02:17:00] flagos(default) | Ascend plugin 首启推理输出全'!'乱码
  详情: --mode flagos 代码注入全量开启绕过 vllm_fl plugin dispatch 的 ascend.yaml flagos_blacklist，
        FlagGems linear/softmax 等在 Ascend910 输出乱码
  操作: 启用 plugin dispatch(VLLM_FL_PREFER_ENABLED=true) +
        固化 ascend.yaml flagos_blacklist(39) + log_ 并集(40算子)到 /etc/environment 重启
  结果: 恢复正常，Paris 推理正确，20 个 FlagGems 算子生效，service_ok=true
```

### Steps to Reproduce
1. 使用代码注入模式启动 vLLM（`--mode flagos`，不启用 plugin dispatch）
2. 全量开启 FlagGems 算子（包括 linear、softmax）
3. 对 reka-flash-3 发起推理请求
4. 观察输出全部为 `!` 字符

### Expected Behavior
FlagGems linear/softmax 算子在 Ascend910 上应该输出正确的推理结果。

### Actual Behavior
输出全部为乱码（`!` 字符），推理完全失败。

### Workaround
使用 plugin dispatch 模式（`VLLM_FL_PREFER_ENABLED=true`），并通过 `ascend.yaml` 的 `flagos_blacklist` 禁用有问题的算子。最终保留 20 个算子可正常工作。

### Additional Context
- 此问题特定于 Ascend910C 平台
- 代码注入模式绕过了 plugin 的黑名单机制，导致已知有问题的算子被强制启用
- 使用 plugin dispatch 后问题解决，说明 plugin 的黑名单配置是准确的

### Possible Directions
1. 修复 FlagGems linear/softmax 在 Ascend 平台的实现
2. 或在代码注入模式下也尊重 plugin 的 flagos_blacklist 配置
3. 在 ascend.yaml 中明确标注这些算子在 Ascend910 上有已知问题
